### DISCUSSION TITLE
Обновление ссылок в документации

### DISCUSSION THREAD

**Discussion by Jampire** [moonshine-software/moonshine#921] [Ideas]
Проект становится всё более популярным. На сторонних ресурсах появляются прямые ссылки на документацию. Но я заметил, что ссылки на документацию были обновлены, а старые ведут в никуда. Предлагаю, в будущем при обновлении на старые ссылки ставить постоянный редирект, чтобы сторонние ресурсы продолжали работать корректно.

**Comment by lee-to** [comment]
Спасибо! Отличное предложение! Если будете находить такие битые ссылки то присылайте сюда или в чат мы будем сразу добавлять редиректы
